src/ui/post_card/small_post_card/small_post_card.tsx

```diff
@@ -0,0 +1,150 @@
+import { styled } from 'styled-components';
+import { PostCardModel } from '../post-card.model';
+import { useState } from 'react';
+
+type SmallPostCardProps = {
+  postCard: PostCardModel;
+};
+
+export const SmallPostCard: React.FC<SmallPostCardProps> = (
+  props: SmallPostCardProps
+) => {
+  const [isLiked, setIsLiked] = useState<boolean>(false);
+  const [isDisliked, setIsDisliked] = useState<boolean>(false);
+  const [amountOfLikes, setAmountOfLikes] = useState<number>(
+    props.postCard.likes_amount
+  );
+  const [isSaved, setIsSaved] = useState<boolean>(false);
+
+  function dislike() {
+    if (isLiked) {
+      setAmountOfLikes(amountOfLikes - 1);
+    }
+    setIsDisliked(!isDisliked);
+    setIsLiked(false);
+  }
+  function like() {
+    setIsLiked(!isLiked);
+    setIsDisliked(false);
+    setAmountOfLikes(isLiked ? amountOfLikes - 1 : amountOfLikes + 1);
+  }
+  return (
+    <SmallPostCardWrapper>
+      <SmallMainWrapper>
+        <SmallCardtextWrapper>
+          <SmallPostCardDate>{props.postCard.date}</SmallPostCardDate>

```

Comment:
check the Intl API - it can help you with converting your data to locale-specific format, e.g.

`12-12-2012` -> `December 12, 2012`